data/hai/hai reveal 4 middle.txt

```diff
@@ -1022,46 +1031,189 @@ mission "Hai Reveal [A09] History of Appeasement"
+				goto questions
+			label childish
+			action
+				set "Hai Reveal [A09] History of Appeasement: interrupted"
+			`	As the pair turn towards you, Teeneep looks mortified. "Captain <last>, I... I'm sorry you had to see me like this. I just... Sayari?"`
+			`	"Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
+				goto arrived
+			label interrupt
+			action
+				set "Hai Reveal [A09] History of Appeasement: interrupted"
+			`	Realizing that you've arrived, Sayari tilts her head slightly to one side to indicate she's speaking to you, while not taking her eyes off Teeneep. "Welcome back, Captain <last>. I must apologize for my friend's state. She's just discovered some rather unfortunate information."`
+				goto arrived
+			label arrived
+			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!`
+			`	"You knew! This whole time, for a half century, you knew, and you did nothing!"`
+			`	Teeneep closes her eyes and uses a deep breathing technique to calm herself down. It looks like she's practiced this many times.`
+			`	The three of you sit down once Teeneep looks ready to talk.`
+				goto questions
+			label friends
+			action
+				set "Hai Reveal [A09] History of Appeasement: label friends"
+			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`
+			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`
+				goto questions
+
+			# Second phase of conversation: find out about the pirates.
+			label deals
+			action
+				set "Hai Reveal [A09] History of Appeasement: label deals"
+			`	Sayari says, "The Elders have been making deals with pirates for a long time, in return for secrecy or ceasing mischef. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've know for quite a while."`
+			`	An enraged Teeneep counters. "They gave them our weapons so that they could more efficiently go and enslave other people rather than just dealing with them properly they way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
```

Comment:
```suggestion
			`	An enraged Teeneep counters. "They give them our weapons so that they can more efficiently go and enslave other people, rather than just dealing with them properly the way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
```

Tense match.